client/scss/main.scss

```diff
@@ -2,6 +2,26 @@
 @import "skeleton";
 @import "leaflet";
 
+body {
+    padding: 50px 0;
+}
+
 #map {
     height: 500px;
+    margin-bottom: 30px;
+}
+
+.container {
+}
+
+.address {
+    text-align: right;
+}
+
+.name {
```

Comment:
may i suggest 'park-name' as a className instead? its descriptive and less likely to cause css conflicts in the future.